.github/workflows/ci.yml

```diff
@@ -32,22 +32,16 @@ jobs:
         include:
           - php: '5.6'
             composer: 1
-            wordpress: '5.9'
-          - php: '7.0'
-            composer: 1
-            wordpress: '5.9'
+            wordpress: '6.2'
           - php: '7.1'
             composer: 1
-            wordpress: '5.9'
-          - php: '7.2'
-            composer: 2
-            wordpress: '5.9'
-          - php: '7.3'
-            composer: 2
-            wordpress: '5.9'
+            wordpress: '6.5'
           - php: '7.4'
             composer: 2
-            wordpress: '5.9'
+            wordpress: '6.6'
+          - php: '8.2'
+            composer: 2
+            wordpress: '6.6'
```

Comment:
Updating the CI setup to test the current major WP version `6` with possible PHP versions.


c.f. 
- https://wordpress.org/documentation/article/supported-versions/
- https://make.wordpress.org/core/handbook/references/php-compatibility-and-wordpress-versions/